camping-omnibus.gemspec-old

```diff
@@ -21,9 +21,15 @@ def camping_spec
     s.email = 'why@ruby-lang.org'
     s.homepage = 'http://camping.rubyforge.org/'
     s.executables = ['camping']
-    s.add_dependency('rack', '>=1.0')
-    s.add_dependency('mab', '>=0.0.3')
-    s.required_ruby_version = '>= 1.8.2'
+    s.add_runtime_dependency('rack', '~> 3.0', '>= 3.0.4.1')
+    s.add_runtime_dependency('rack-session', '~> 2.0', '>=2.0.0')
+    s.add_runtime_dependency('mab', '~> 0.0', '>=0.0.3')
+
+    # Post Rack 3.0 dependencies.
+    # s.add_runtime_dependency('rack', '~> 3.0', '>= 3.0.0')
```

Comment:
Do these need to be active or can these comments be removed?

Reply:
No, I simplified this, removed the comments.